### ISSUE TITLE
Question on influence computation

### ISSUE THREAD

**Issue by yushundong** [nimarb/pytorch_influence_functions#30]
Hi, thanks for your amazing contribution of this Pytorch implementation!

Question on influence computation: as mentioned in function calc_s_test_single: s_test = invHessian * nabla(Loss(test_img, model params). However, as in Eq. (2) in the paper Understanding Black-box Predictions via Influence Functions, we should compute invHessian * nabla(Loss(train_img, model params). It seems that the positions of train and test image in Eq. (2) is switched. 

Did I get anything wrong? Could you please provide more clue on that? Thanks so much for your information!

**Comment by lange-martin** [comment]
Have a look at chapter 3 in the paper: "Efficiently calculating influence". Here, s_test is definied as invHessian * nabla(Loss(test_img, params)).
